jdbc/src/test/java/org/apache/metamodel/jdbc/dialects/OracleQueryRewriterTest.java

```diff
@@ -92,6 +92,14 @@ public void testOffsetFetchVersionCheck() throws SQLException {
         Assert.assertEquals("The query shouldn't be rewritten.", query.toSql(), qr.rewriteQuery(query));
     }
 
+    @Test
+    public void testOffsetFetchVersionSpecialCaseCheck() throws SQLException {
```

Comment:
The test case here seems to rely on a number of other factors and as such I think the assertion is a bit of a by-product of the version parsing logic, not actually testing the parsing itself.

Reply:
I would delete this test case now.